<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>muroaserverClass</class>
 <widget class="QMainWindow" name="muroaserverClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Muroa Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Playlist:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>595</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="command_label">
      <property name="text">
       <string>&lt;no command running&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Response:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLabel" name="response_label">
      <property name="text">
       <string>&lt;none&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Log:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>589</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="action_Open"/>
   </widget>
   <widget class="QMenu" name="menu_Test">
    <property name="title">
     <string>&amp;Test</string>
    </property>
    <addaction name="action_next_Revision"/>
    <addaction name="actionScan_collection"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Test"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open ...</string>
   </property>
  </action>
  <action name="action_next_Revision">
   <property name="text">
    <string>&amp;next Revision</string>
   </property>
  </action>
  <action name="actionScan_collection">
   <property name="text">
    <string>scan collection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
